docs/source/howto/user_interface.rst

```diff
@@ -25,11 +26,51 @@ To switch back to the classic notebook, add ``/tree`` to the URL like so:
 
    http(s)://<server:port>/tree
 
+For example, the following Binder URL will open the
+`pyTudes repository <https://github.com/norvig/pytudes>`_
+and begin a JupyterLab session in the ``ipynb`` folder:
+
+https://mybinder.org/v2/gh/norvig/pytudes/master?urlpath=lab/tree/ipynb
+
+The ``/tree/ipynb`` above is how JupyterLab directs you to a specific file
+or folder.
+
 To learn more about URLs in JupyterLab and Jupyter Notebook, visit
 `starting JupyterLab <http://jupyterlab.readthedocs.io/en/latest/getting_started/starting.html>`_.
 
+
+Nteract
+=======
+
+`Nteract is a notebook interface <https://nteract.io/>`_ built with React.
+It is similar to a more feature-filled version of the traditional
+Jupyter Notebook interface.
+
+Nteract comes pre-installed in any session that has been built from
```

Comment:
I think `nteract` is lower-case in most places in those docs - so might be better to use lower-case for consistency?

Reply:
Yup, that's the case.